### PR TITLE
Feat/filterpage

### PR DESCRIPTION
**Pull request by KatharinaWuerth**

### CODE REVIEW

File: src/filterPage/ButtonList.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import OptionButton from './OptionButton'
+import { optionList } from '../app/App'
+import { NavLink } from 'react-router-dom'
+
+const StyledOptionButtonList = styled.section`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```
Comment (akdrewes) [inline src/filterPage/ButtonList.js]: You could also write: grid-template-columns: repeat(2, 1fr).

File: src/app/App.js [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 
 let plants = plantObjects
 
-const GridPlantList = styled.div`
-  display: grid;
-  grid-template-rows: 50px 1fr;
-  height: 100vh;
-  background: linear-gradient(0deg, white 70%, #367860 30%);
-`
+let optionList = [
```
Comment (akdrewes) [inline src/app/App.js]: Better using const instead of let, if the list doesn't need to be changed.

File: src/filterPage/ButtonList.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  grid-template-rows: auto;
+  padding: 20px;
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
```
Comment (akdrewes) [inline src/filterPage/ButtonList.js]: As grid-column-gap and grid-row-gap have the same value, you could replace these two lines with grid-gap: 20px.

File: src/app/App.js [hunk 4]
```diff
@@ -1,39 +1,132 @@
...
+    if (id.includes('not')) {
+      const partId = id.split('-')[1]
+      const filtered = selection.filter(item => item !== partId)
+      setSelection([...filtered, id])
+      return
+    }
+    // remove possible 'not'-item + add item
+    const filtered = selection.filter(item => item !== `not-${id}`)
+    setSelection([...filtered, id])
   }
```
Comment (AntoniaKro) [inline src/app/App.js]: sehr tricky, aber logisch

File: src/filterPage/FilterPage.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  line-height: 1.5;
+  font-family: 'Raleway', sans-serif;
+  font-size: 1.5rem;
+  text-align: center;
+`
+
+export default function FilterPage({ options, selection, onOptionSelect }) {
+  return (
+    <GridFilterPage>
+      <GlobalStyles />
```
Comment (AntoniaKro) [inline src/filterPage/FilterPage.js]: I think Global Styles is only necessary once in the App.

File: src/app/App.js [hunk 6]
```diff
@@ -1,39 +1,132 @@
...
+            }
+            onBookmark={handleBookmark}
+            {...props}
+          />
+        )}
+      />
+    </BrowserRouter>
   )
+
+  function getFilteredPlants() {
```
Comment (AntoniaKro) [inline src/app/App.js]: Why are these function below the JSX? Maybe lift it above the return

**Pull request merged**